ci/gitlab-ci/deploy-aws/.gitlab-ci.yml

```diff
@@ -0,0 +1,15 @@
+image: alpine
+
+stages:
+    - deploy
+
+deploy:
+    stage: deploy
+    variables:
+      AWS_REGION: us-east-1
+      AWS_ACCESS_KEY_ID: secrethub://company/app/aws/access_key_id
+      AWS_SECRET_ACCESS_KEY: secrethub://company/app/aws/secret_access_key
+    before_script:
+        - apk add --repository https://alpine.secrethub.io/alpine/edge/main --allow-untrusted secrethub-cli
+    script:
+        - secrethub run -- ./deploy.sh
```

Comment:
I'm not sure how relevant or useful this example is, but I thought of adding it for completeness since it is used in the guide.

Reply:
I'd say: let's not add examples unless they can actually be executed. So either use the AWS CLI to make it work or remove it for now.

Reply:
Yes, or create a `deploy.sh` file

Reply:
I added the `deploy.sh` file